### ISSUE TITLE
Conflicting MEME results between datamonkey.org and classic.datamonkey.org

### ISSUE THREAD

**Issue by nmathers** [veg/hyphy#814]
When I an input file of 80 sequences with a user tree to MEME on the classic datamonkey.org site I get a result of 23 sites with evidence of EDS at p < 0.1 significance level.  However, when I submit the identical input to the new datamonkey.org site, the result is 0 sites with evidence of EDS at p < 0.1.

Are the two sites running different versions of MEME? Or is this an error?

Thanks a lot!

**Comment by spond** [comment]
Dear @nmathers,

This behavior definitely merits further investigation. Could you either post or email me the URLs for the two respective analyses?

Best,
Sergei

**Comment by spond** [comment]
Dear @nmathers,

I have looked into the issue and believe the modern datamonkey result is likely correct. First, however, I have realized that Hyphy was not recognizing the tree included in your NEXUS file (not 100% certain why, but I think it has to do with the file's use of the deprecated 'data' block).

With the user tree, modern datamonkey shows 1 site under selection (http://www.datamonkey.org/meme/5b3a30fc0ba11710eda255c3), but this is still quite different from the classic result.

Occasionally, MEME can experience convergence problems at individual sites, and I believe this is what might have happened here.  Modern MEME achieves better log likelihoods at all of the sites that were identified as selected by the old MEME. The modern MEME version uses better heuristics (I believe) to initialize site optimizations which tend to converge faster and more accurately.

As a sanity check, it always helps to also look at other methods (like FUBAR and FEL) to see whether or not at least some of the MEME sites "light up", which they don't on your data.

Best,
Sergei
